src/pages/ProfilePage.jsx

```diff
@@ -1,4 +1,39 @@
 /** @jsx createVNode */
+import { Footer, Header, Navigation } from "../components";
 import{ createVNode } from "../lib";
+import { globalStore } from "../stores";
 
-export const ProfilePage = () => ({});
+
+export const ProfilePage = () => {
+  const { loggedIn, currentUser } = globalStore.getState();
+  const { username = '', email = '', bio = '' } = currentUser ?? {}
+  return (
+    <div class="bg-gray-100 min-h-screen flex justify-center">
+      <div class="max-w-md w-full">
+        {Header()}
+        {Navigation({ loggedIn })}
```

Comment:
여기도 컴포넌트 처럼 표현하면 좋았을 것 같아요!
```suggestion
        {<Header />}
        {<Navigation loggedIn={loggedIn} />}
```